Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -777,12 +786,11 @@ class CombinatorialKalmanFilter {
         const auto& sourceLink = *it;
 
         // prepare the track state
-        PM mask =
-            PM::Predicted | PM::Jacobian | PM::Uncalibrated | PM::Calibrated;
+        PM mask = PM::Predicted | PM::Jacobian | PM::Calibrated;
 
         if (it != slBegin) {
           // not the first TrackState, only need uncalibrated and calibrated
-          mask = PM::Uncalibrated | PM::Calibrated;
+          mask = PM::Calibrated;
```

Comment:
As discussed in MM, `Uncalibrated` becomes *always allocated*, so is removed from the enum. The CKF then doesn't have to tell MTJ to allocated `Uncalibrated`, because it's allocated by default.